dotty-community-build/src/test/scala/dotty/communitybuild/CommunityProjectTest.scala

```diff
@@ -85,9 +85,6 @@ abstract class CommunityProjectTest(project: CommunityProject) {
       val exit = process.waitFor()
       assertEquals(exit, 0)
     }
-    exec("git", "clean", "-xfd")
-    exec("git", "checkout", "dotty")
-    exec("git", "pull", "origin", "dotty")
```

Comment:
Theses don't play nice with git submodules. I would either get rid of submodules and just clone the repo to build in the test or get rid of these lines. WDYT @olafurpg?

Reply:
I agree this is suboptimal. What do you think about replacing submodules with jgit to clone/checkout?

Reply:
I agree. I will address it in another PR